tekdoc/tekdoc_test.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"bytes"
+	"github.com/google/go-cmp/cmp"
+	"testing"
+)
+
+func TestTekdoc(t *testing.T) {
+	task, err := read("buildkit-daemonless.yaml")
+	if err != nil {
+		t.Fatal(err)
```

Comment:
Include a short error message about what failed (e.g. `t.Fatalf("failed to read task: %v", err)`)